### PR TITLE
Boavizta test coverage

### PR DESCRIPTION
**Pull request by manushak**
### Types of changes

- [x] Enhancement (project structure, spelling, grammar, formatting)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### A description of the changes proposed in the Pull Request

- add full test coverage for boavizta
- make some small imporvents

### CODE REVIEW

File: src/__tests__/unit/lib/boavizta/index.test.ts [hunk 1]
```diff
@@ -100,6 +129,36 @@ describe('lib/boavizta: ', () => {
         expect(await output.execute([])).toEqual([]);
       });
 
+      it('throws an error when the metric type is `gpu-util`.', async () => {
```
Comment (narekhovhannisyan) [inline src/__tests__/unit/lib/boavizta/index.test.ts:132]: title says it's throws an error, but the implementation says it has result

File: src/__tests__/unit/lib/boavizta/index.test.ts [hunk 2]
```diff
@@ -188,6 +247,115 @@ describe('lib/boavizta: ', () => {
...
+              timestamp: '2021-01-01T00:00:00Z',
+              duration: 15,
+              'cpu/utilization': 34,
+              'instance-type': 't2.micro',
+              country: 'USA',
+              provider: 'aws1',
+            },
+          ]);
+        } catch (error) {
+          expect(error).toBeInstanceOf(UnsupportedValueError);
```
Comment (narekhovhannisyan) [inline src/__tests__/unit/lib/boavizta/index.test.ts:266]: please also assert the error message if possible

File: src/__tests__/unit/lib/boavizta/index.test.ts [hunk 3]
```diff
@@ -64,6 +64,35 @@ describe('lib/boavizta: ', () => {
         ]);
       });
 
+      it('returns a result when the impacts from the API is undefined.', async () => {
```
Comment (narekhovhannisyan) [inline src/__tests__/unit/lib/boavizta/index.test.ts:67]: why we are returning valid result if the API response is undefined?

File: src/__tests__/unit/lib/boavizta/index.test.ts (hunk 2 above)
Comment (manushak) [reply src/__tests__/unit/lib/boavizta/index.test.ts:266]: done

File: src/__tests__/unit/lib/boavizta/index.test.ts (hunk 1 above)
Comment (manushak) [reply src/__tests__/unit/lib/boavizta/index.test.ts]: changed

File: src/__tests__/unit/lib/boavizta/index.test.ts (hunk 3 above)
Comment (manushak) [reply src/__tests__/unit/lib/boavizta/index.test.ts]: updated description. We set it to 0 in case of there isn't data from the API

**Review by MariamKhalatova — APPROVED** [review]

**Pull request merged**